.github/workflows/deploy-java-app.yml

```
@@ -23,6 +23,10 @@ on:
         type: string
         description: The name of a GitHub deployment environment in the repository that determines which deployment protection rules, such as reviews, will apply to this deployment.
         required: false
+      environment-type:
+        type: string
+        description: Where the java is being deployed to. 'prod' uses larger-runner-java-prod; 'nonprod' uses larger-runner-java-nonprod.
```

We're deploying a java app, not java itself.
```suggestion
        description: The environment type to which this workflow will deploy the java application. The workflow will choose an appropriate runner that has access to the deployment environment. Either 'prod' or 'nonprod'.
```